spikeinterface/comparison/groundtruthstudy.py

```diff
@@ -247,7 +247,9 @@ def compute_metrics(self, rec_name, metric_names=['snr'],
 
         # metrics
         metrics = compute_quality_metrics(we, metric_names=metric_names)
-        filename = self.study_folder / 'metrics' / f'metrics _{rec_name}.txt'
+        folder = self.study_folder / 'metrics'
+        folder.mkdir()
```

Comment:
Should this have an `exist_ok=True`? Or the assumption that it is always non existent is ok?

Reply:
right, better with an exist_ok flag. Pushing now